joomla/joomla-cms
# [4.2] Mail Templates htmlbody

### Summary of Changes
html emails are much much bigger than plain text emails. Using some email templates from beefree.io was almost impossible as they were all too big to be saved. Checking the database the field type was TEXT and yet for the fulltext field in #__content MEDIUMTEXT was used. So it wasnt even possible to use an entire article in the mail template.

This PR changes the field to MEDIUMTEXT in mysql for updates and new installations. Note postgres doesnt need any change as it only has one datatype for text.


### Testing Instructions
Upgrade with the prebuilt test packages and check the datatype of the htmlbody field in the #__mail_templates table
Clean install with he prebuilt test packages and check the datatype of the htmlbody field in the #__mail_templates table

check of the database checker in the system dashboard. 
. In case of an update to the patched package for the PR, the checker should only show the usual error about not matching CMS version when using such packages, and in case of a new installation no errors should be shown.



### Actual result BEFORE applying this Pull Request
TEXT


### Expected result AFTER applying this Pull Request
MEDIUMTEXT


### Documentation Changes Required
none
